### PR TITLE
FIX: User status tooltip flaky test

### PR DESCRIPTION
**Pull request by jancernik**

### CODE REVIEW

**Comment by CvX** [comment]
Looks like it's still flaky 😅

**Comment by jancernik** [comment]
yeah.... :roll_eyes:

**Comment by jancernik** [comment]
@CvX now it's for real it seems

**Comment by CvX** [comment]
You'll need to merge current `main` and uncomment recently skipped tests to confirm it 100% 😃

File: app/assets/javascripts/discourse/tests/acceptance/post-inline-mentions-test.js [hunk 1]
```diff
@@ -36,12 +36,12 @@ async function mouseenter() {
...
 
-async function mouseleave() {
-  await triggerEvent(query(".user-status-message"), "mouseleave");
-}
-
 acceptance("Post inline mentions", function (needs) {
   needs.user();
+  needs.hooks.afterEach(() => {
+    const tippy = document.querySelector("[data-tippy-root]");
+    tippy && document.querySelector("body").removeChild(tippy);
```
Comment (CvX) [inline app/assets/javascripts/discourse/tests/acceptance/post-inline-mentions-test.js:43]: I don't see other tippy-related tests doing any cleanup 🤔 Perhaps this particular component (or whatever is creating it 😉) is missing `tippyInstance.destroy()`?

Comment (jancernik) [reply app/assets/javascripts/discourse/tests/acceptance/post-inline-mentions-test.js:43]: I don't think so. We were really careful about using `tippyInstance.destroy()` everywhere it was needed.
The function responsible for this one

https://github.com/discourse/discourse/blob/148431d52182d9c0810536bc12a76caee09c2e0c/app/assets/javascripts/discourse/app/widgets/post-cooked.js#L385-L389

It gets called from the `destroy` function in the widget.

AFAIK all the other tippy-related are Ember components tests. These render tests seem to decommission the component better in between tests. I checked, and `instance.destroy()` gets called every time the test finishes, but it doesn't seem to have an effect in this environment. Manually testing this, it works as expected.

The mouse leave() function was to force this state, but it seems it was not working reliably.

**Comment by jjaffeux** [comment]
@AndrewPrigorshnev @CvX is this related to what you are working on?

**Comment by jjaffeux** [comment]
Supersed by https://github.com/discourse/discourse/pull/22435

**Pull request closed without merging**